client/my-sites/stats/stats-module-utm/index.jsx

```diff
@@ -12,11 +14,14 @@ const StatsModuleUTMWrapper = ( { siteId, period, postId, query, summary, classN
 
 	// Check if blog is internal.
 	const { isPending: isFetchingUsage, data: usageData } = usePlanUsageQuery( siteId );
+	// Check if the UTM stats module is supported by standalone Stats purchases.
```

Comment:
The following line checks if the commercial plan is owned, not UTM.

Reply:
Yes, the note here is trying to describe a different type of support for advanced features, like UTM stats, since Atomic sites would have features bundled in Dotcom plans rather than a standalone Stats commercial license.